test/unit/test_mapping.py

```diff
@@ -204,7 +204,7 @@ def test_env_vars_round_trip():
     assert env_vars['SM_HPS'] == '{"batch_size":64,"epochs":10,"loss":"SGD","precision":5.434322}'
     assert env_vars['SM_HP_PRECISION'] == '5.434322'
     assert env_vars['SM_RESOURCE_CONFIG'] == '{"current_host":"algo-1","hosts":["algo-1","algo-2","algo-3"]}'
-    assert env_vars['SM_MODULE_NAME'] == 'user_script'
+    assert env_vars['SM_USER_PROGRAM'] == 'user_script.py'
```

Comment:
isn't it a breaking change?
what if someone expects the old environment variable?